projects/js-packages/charts/src/components/leaderboard-chart/leaderboard-chart.module.scss

```diff
@@ -0,0 +1,101 @@
+.leaderboardChart {
+	width: 100%;
+	max-width: 400px;
+	display: flex;
+	flex-direction: column;
+	gap: 12px;
```

Comment:
If we are going to begin using HTML+CSS to compose the chart (as we do here), we should also consider using WordPress components. The regular div with these styles could be replaced with the `<VStack />` component, for instance.

I'd like to know the opinion of other Chart development folks here, though.

Reply:
I agree with Damián.

I noticed that the Chart package doesn't currently use any WordPress components, perhaps we should check with @kangzj to confirm.

Reply:
> The regular div with these styles could be replaced with the <VStack /> component, for instance.

Yes, I struggled with that, but I turned the draft into a PR for more solid feedback. We should reuse the core component as much as possible to avoid design deviations.

> I noticed that the Chart package doesn't currently use any WordPress components

That might deserve a confirmation about whether we plan to introduce more bundle sizes for the library. 🙂

Reply:
We should probably try to use as less dependencies as possible; however Core components are used in almost all of our projects, and if consumers are not using HTML based charts, they could be shook off by their bundlers, so I think it's fine to add it.

Reply:
Sounds thoughtful and reasonable! 💯